bedrock/mozorg/templates/mozorg/home/home-new.html

```diff
@@ -23,6 +23,7 @@
 {% block page_desc %}{{ ftl('home-did-you-know-mozilla-the-maker') }}{% endblock %}
 
 {% set show_fundraising_banner = switch('fundraising-banner-july2024', ["en", "de", "fr"]) %}
+{% set show_fundraising_nav_cta = switch('fundraising-nav-cta-july2024'), ["en"] %}
```

Comment:
**issue:** the lang for this switch isn't quite set correctly:

```suggestion
{% set show_fundraising_nav_cta = switch('fundraising-nav-cta-july2024', ["en"]) %}
```

Reply:
**suggestion:** when this merges on dev, and all switches default to `True`, this is gonna be weird as there will be two experiments both trying to do redirects using the same variation variables. I'd suggest updating the logic here to ensure only one thing is running at the same time. Does that make sense?

```suggestion
{% set show_fundraising_nav_cta = switch('fundraising-nav-cta-july2024', ["en"]) and not show_fundraising_banner %}
```

Reply:
Good thinking 👍🏼